### PR TITLE
v2 -> v1 plugin

### PR DESCRIPTION
**Pull request by TahaAttari**

### CODE REVIEW

File: external/pom.xml [hunk 1]
```diff
@@ -71,6 +71,11 @@
 			<groupId>org.opencds.cqf.cql</groupId>
 			<artifactId>evaluator.fhir</artifactId>
 		</dependency>
+		<!-- <dependency>
+			<groupId>com.converter</groupId>
+			<artifactId>cqf-ruler-v1-converter-plugin</artifactId>
+			<version>1.0.0</version>
+		</dependency> -->
```
Comment (TahaAttari) [inline external/pom.xml:78]: without adding the plugin as a dependency to the server I can't import it to Application.java, but if I import it then I get a circular dependency error

**Comment by TahaAttari** [comment]
Planning to rename to `$ersd-v2-to-v1-transform` as per the ticket

File: pom.xml [hunk 2]
```diff
@@ -62,7 +62,6 @@
 	</developers>
 
 	<modules>
-		<module>example</module>
```
Comment (TahaAttari) [inline pom.xml:65]: removed `example` from the modules so that it's obvious that you need to run `mvn package` inside the `example` folder separately

**Review by sliver007 — APPROVED** [review]
Looks good. Builds, the tests pass and I have tested manually and validated the results.

**Pull request merged**